front/src/components/UI/CompanyView/CompaniesSector.tsx

```diff
@@ -0,0 +1,22 @@
+import CompanyBlock from "./CompanyBlock";
+
+const Companies = (props: any) => {
+  return (
+    <div>
```

Comment:
przy takim rozwiązaniu jeśli pojawi się w tablicy z przykładami kolejny item to niestety nie zostanie wyświetlony...

Zamiast tego powinieneś uzyc tutaj podejścia z wykorzystaniem funkcji .map wywoływanej na tablicy. Wtedy dynamicznie będą tworzyć się kolejne elementy jeśli zmienimy coś w liście